uplink/builder.py

```diff
@@ -160,21 +160,13 @@ def __init__(
             hook=None,
             converter_factories=()
     ):
-        builder = Builder()
-        builder.base_url = base_url
-        builder.add_converter_factory(*converter_factories)
+        self.builder = Builder()
```

Comment:
Could you set `self._builder` instead. While python doesn't have a concept of `private` and `public` attributes/methods,  I think it's good practice to use a single leading underscore to omit it from the public API.